packages/block-library/src/text-columns/block.json

```diff
@@ -0,0 +1,29 @@
+{
+	"name": "core/text-columns",
+	"icon": "columns",
+	"category": "layout",
+	"attributes": {
+		"content": {
+			"type": "array",
+			"source": "query",
+			"selector": "p",
+			"query": {
+				"children": {
+					"type": "string",
+					"source": "html"
+				}
+			},
+			"default": [ { }, { } ]
+		},
+		"columns": {
+			"type": "number",
+			"default": 2
+		},
+		"width": {
+			"type": "string"
+		}
+	},
+	"supports": {
+		"inserter": false
+	}
```

Comment:
I feel like I have a lot to say about these :), can we start with something very small like the "category"?

Reply:
I know this PR is more about the way javascript consumes this file rather than about the content of the file directly and that's why I suggest to keep it to the minimum for the moment.

Reply:
I don't care what we include, I opted for everything that server is able to expose to the client:

https://github.com/WordPress/wordpress-develop/blob/689ba4eec66b8d735c57940303ea0d49657f4cb3/src/wp-admin/includes/post.php#L2206

Reply:
Agreed it's not very important for now (to filter or not) but the shape of some of these properties might need to be changed later.

Reply:
`supports` removed, should I remove anything else?